source/usage-examples/replaceOne.txt

```diff
@@ -0,0 +1,46 @@
+==================
+Replace a Document
+==================
+
+.. default-domain:: mongodb
+
+You can replace a single document using the `replaceOne()
+<https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#replaceOne>`_
+method. ``replaceOne()`` accepts a query document and a replacement
+document, and replaces the first document that matches the filter with
+the provided replacement document.
+
+You can specify additional options, such as ``upsert``, using the
+optional ``options`` parameter. Configuring the ``upsert`` field to
+``true`` in the options object causes the operation to insert a new
+document if no document matches the query.
+
+The ``replaceOne()`` method returns a `Promise
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise>`_
+that resolves to an `updateWriteOpResult
+<https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#~updateWriteOpResult>`_.
```

Comment:
To me, the sentences that follow would be more clear if you mention that the `updateWriteOpResult` is a document that contains the following fields.

Reply:
Clarified.